net-libs/webkit-gtk/webkit-gtk-2.41.6-r600.ebuild

```diff
@@ -0,0 +1,251 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_REQ_USE="xml(+)"
+PYTHON_COMPAT=( python3_{9..12} )
+USE_RUBY="ruby30 ruby31 ruby32"
+
+inherit check-reqs flag-o-matic gnome2 optfeature python-any-r1 ruby-single toolchain-funcs cmake
+
+MY_P="webkitgtk-${PV}"
+DESCRIPTION="Open source web browser engine"
+HOMEPAGE="https://www.webkitgtk.org"
+SRC_URI="https://www.webkitgtk.org/releases/${MY_P}.tar.xz"
+
+LICENSE="LGPL-2+ BSD"
+SLOT="6/0" # soname version of libwebkit2gtk-6.0
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+IUSE="aqua avif examples gamepad gles2-only keyring +gstreamer +introspection pdf +jpeg2k +jumbo-build lcms seccomp spell systemd wayland X"
```

Comment:
I think we have to remove `IUSE=gles2-only` since the configuration option is gone.

I guess this is the result of these items mentioned in the NEWS file:

```diff
+  - Remove support for OpenGL API in the web process.
```

```diff
+  - Remove support for rendering with GLX in the web process.
```

Probably continuing to transition to ANGLE.